app/src/main/java/org/lineageos/selfie/MainActivity.kt

```diff
@@ -556,6 +572,18 @@ class MainActivity : AppCompatActivity() {
         bindCameraUseCases()
     }
 
+    /**
+     * Update grid aspect ratio based on current preview size
+     */
+    @androidx.camera.view.TransformExperimental
+    private fun updateGridAspectRatio() {
+        val outputTransform = viewFinder.outputTransform ?: return
+        val size = outputTransform.getViewPortSizeUnrestricted()
+
+        // For somehow reason, height and width are inverted :D
+        gridView.previewSize = Size(size.height, size.width)
```

Comment:
It's likely the case it uses landscape

Reply:
It's inverted on all the devices I've tested with